zookeeper-server/src/main/java/org/apache/zookeeper/server/DataTree.java

```diff
@@ -1521,4 +1562,179 @@ public boolean removeWatch(String path, WatcherType type, Watcher watcher) {
     public ReferenceCountedACLCache getReferenceCountedAclCache() {
         return aclCache;
     }
+
+    /**
+     * Add the digest to the historical list, and update the latest zxid digest.
+     */
+    private void logZxidDigest(long zxid, long digest) {
+        ZxidDigest zxidDigest = new ZxidDigest(zxid, DigestCalculator.DIGEST_VERSION, digest);
+        lastProcessedZxidDigest = zxidDigest;
+        if (zxidDigest.zxid % 128 == 0) {
```

Comment:
Can you explain this magic value '128' ? Maybe a comment will help.

Maybe I am missing something

Reply:
Better than put a comment it is to extract as a constant...